lib/types/cql-utils.js

```diff
@@ -101,6 +101,21 @@ function ensureNumber(value) {
     }
 }
 
+/**
+ * Encodes tuple into QueryParameterWrapper
+ * @param {types.Tuple} value
+ * @param {rust.ComplexType} complexType
+ * @returns {Array<rust.QueryParameterWrapper>}
+ */
+function encodeTuple(value, complexType) {
+    let res = [];
+    for (let i = 0; i < value.length; i++) {
+        res.push(getWrapped(complexType.getFirstSupportType(), value.get(i)));
+        complexType = complexType.getSecondSupportType();
+    }
```

Comment:
Please, don't. Don't implement tuple as a linked list... Ferris is already crying. I need to calm him down all the time.